src/drivers/eeprom_i2c.h

```diff
@@ -0,0 +1,10 @@
+#include "simba.h"

```

Comment:
- Missing file header.
- Missing double inclusion guard.
- Missing function comments.
- Fix indentation.
- Move ``eeprom_i2c_module_init()`` before any other function, then ``eeprom_i2c_init()`` and then read and write.